Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -862,14 +862,13 @@ class BrowserViewController: UIViewController {
 
     override func observeValueForKeyPath(keyPath: String?, ofObject object: AnyObject?, change: [String: AnyObject]?, context: UnsafeMutablePointer<Void>) {
         let webView = object as! WKWebView
-        if webView !== tabManager.selectedTab?.webView {
-            return
-        }
         guard let path = keyPath else { assertionFailure("Unhandled KVO key: \(keyPath)"); return }
         switch path {
         case KVOEstimatedProgress:
-            guard let progress = change?[NSKeyValueChangeNewKey] as? Float else { break }
-            urlBar.updateProgressBar(progress)
+            if webView == tabManager.selectedTab?.webView {
+                guard let progress = change?[NSKeyValueChangeNewKey] as? Float else { break }
```

Comment:
Is there a better way of writing this guard statement?
- looks like commas are allowed only if there are multiple let statements
- can't put the let statement within parenthesis
- I'm not happy with the guard statement below because there where clause is not related to the primary let statement

 ```
           guard let progress = change?[NSKeyValueChangeNewKey] as? Float where (webView == tabManager.selectedTab?.webView) else {
                break
            }

```